### ISSUE TITLE
3 OPs are not supported yet when translating model from tensorflow to paddle

### ISSUE THREAD

**Issue by xttian** [PaddlePaddle/X2Paddle#386]
Notes:
Now translating model from tensorflow to paddle.
Total nodes: 4129
Converting node 4129 ...     ========= 3 OPs are not supported yet ===========
========== NotEqual ============
========== Unpack ============
========== Square ============

The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.

**Comment by jiangjiajun** [comment]
> Notes:
> Now translating model from tensorflow to paddle.
> Total nodes: 4129
> Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> ========== NotEqual ============
> ========== Unpack ============
> ========== Square ============
> 
> The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.

Could you provide your tensorflow model for us to debug? Please send to jiangjiajun@baidu.com, thks

**Comment by xttian** [comment]
> jiangjiajun@baidu.com

Of course, I have accomplished it. Please check it.  E-mail: 188xxxx7285@163.com

**Comment by jiangjiajun** [comment]
> > Notes:
> > Now translating model from tensorflow to paddle.
> > Total nodes: 4129
> > Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> > ========== NotEqual ============
> > ========== Unpack ============
> > ========== Square ============
> > The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.
> 
> Could you provide your tensorflow model for us to debug? Please send to [jiangjiajun@baidu.com](mailto:jiangjiajun@baidu.com), thks

你好，请先卸载现在的x2paddle，按照如下方式安装x2paddle后进行转换
```
git clone https://github.com/PaddlePaddle/X2Paddle.git
cd X2Paddle
git checkout develop_for_issue_386
python setup.py install
```
目前并未验证转换的正确性，如有问题请在此issue下继续反馈

**Comment by xttian** [comment]
> > > Notes:
> > > Now translating model from tensorflow to paddle.
> > > Total nodes: 4129
> > > Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> > > ========== NotEqual ============
> > > ========== Unpack ============
> > > ========== Square ============
> > > The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.
> > 
> > 
> > Could you provide your tensorflow model for us to debug? Please send to [jiangjiajun@baidu.com](mailto:jiangjiajun@baidu.com), thks
> 
> 你好，请先卸载现在的x2paddle，按照如下方式安装x2paddle后进行转换
> 
> ```
> git clone https://github.com/PaddlePaddle/X2Paddle.git
> cd X2Paddle
> git checkout develop_for_issue_386
> python setup.py install
> ```
> 
> 目前并未验证转换的正确性，如有问题请在此issue下继续反馈

问题没有解决，报错信息和原来一样
Converting node 4129 ...     ========= 3 OPs are not supported yet ===========
========== Unpack ============
========== Square ============
========== NotEqual ============

**Comment by jiangjiajun** [comment]
> > > > Notes:
> > > > Now translating model from tensorflow to paddle.
> > > > Total nodes: 4129
> > > > Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> > > > ========== NotEqual ============
> > > > ========== Unpack ============
> > > > ========== Square ============
> > > > The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.
> > > 
> > > 
> > > Could you provide your tensorflow model for us to debug? Please send to [jiangjiajun@baidu.com](mailto:jiangjiajun@baidu.com), thks
> > 
> > 
> > 你好，请先卸载现在的x2paddle，按照如下方式安装x2paddle后进行转换
> > ```
> > git clone https://github.com/PaddlePaddle/X2Paddle.git
> > cd X2Paddle
> > git checkout develop_for_issue_386
> > python setup.py install
> > ```
> > 
> > 
> > 目前并未验证转换的正确性，如有问题请在此issue下继续反馈
> 
> 问题没有解决，报错信息和原来一样
> Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> ========== Unpack ============
> ========== Square ============
> ========== NotEqual ============

不好意思，发现代码并未更新上去，请再重新拉代码安装，进行转换

**Comment by xttian** [comment]
> > > > > Notes:
> > > > > Now translating model from tensorflow to paddle.
> > > > > Total nodes: 4129
> > > > > Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> > > > > ========== NotEqual ============
> > > > > ========== Unpack ============
> > > > > ========== Square ============
> > > > > The tf model is trained for text classification, where there is a pretrained bert layer and some CNN layers.
> > > > 
> > > > 
> > > > Could you provide your tensorflow model for us to debug? Please send to [jiangjiajun@baidu.com](mailto:jiangjiajun@baidu.com), thks
> > > 
> > > 
> > > 你好，请先卸载现在的x2paddle，按照如下方式安装x2paddle后进行转换
> > > ```
> > > git clone https://github.com/PaddlePaddle/X2Paddle.git
> > > cd X2Paddle
> > > git checkout develop_for_issue_386
> > > python setup.py install
> > > ```
> > > 
> > > 
> > > 目前并未验证转换的正确性，如有问题请在此issue下继续反馈
> > 
> > 
> > 问题没有解决，报错信息和原来一样
> > Converting node 4129 ... ========= 3 OPs are not supported yet ===========
> > ========== Unpack ============
> > ========== Square ============
> > ========== NotEqual ============
> 
> 不好意思，发现代码并未更新上去，请再重新拉代码安装，进行转换

～$ git clone https://github.com/PaddlePaddle/X2Paddle.git
Cloning into 'X2Paddle'...
remote: Enumerating objects: 226, done.
remote: Counting objects: 100% (226/226), done.
remote: Compressing objects: 100% (152/152), done.
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
一直clone不下来，是文件太大了吗？我之前设置了git 500M，之前的可以下载，现在不行了

**Comment by xttian** [comment]
> develop_for_issue_386

我重新更新之后，问题没有解决，还是一样的报错

**Comment by jiangjiajun** [comment]
> > develop_for_issue_386
> 
> 我重新更新之后，问题没有解决，还是一样的报错

先卸载现有的x2paddle，再拉最新的代码，checkout至develop_for_issue_386，安装即可。我这边已经测试过可以了，建议您再检查下操作步骤。

**Comment by xttian** [comment]
> > > develop_for_issue_386
> > 
> > 
> > 我重新更新之后，问题没有解决，还是一样的报错
> 
> 先卸载现有的x2paddle，再拉最新的代码，checkout至develop_for_issue_386，安装即可。我这边已经测试过可以了，建议您再检查下操作步骤。

好的，我新建一个环境试一下
